types/types.go

```diff
@@ -56,12 +56,15 @@ type Image interface {
 // ImageManifest is the interesting subset of metadata about an Image.
 // TODO(runcom)
 type ImageManifest interface {
+	Raw() []byte
+	Layers() []string
 	String() string
 }
 
 // DockerImageManifest is a set of metadata describing Docker images and their manifest.json files.
 // Note that this is not exactly manifest.json, e.g. some fields have been added.
 type DockerImageManifest struct {
```

Comment:
agree - they're there since the beginning and are only used when pretty-printing to the cli (after an inspect command) - though - those fields are inside this struct, most of the time I'd like to deal with image manifests struct via the interface above